pandas/io/excel.py

```diff
@@ -1621,10 +1629,10 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0,
             if cell.mergestart is not None and cell.mergeend is not None:
 
                 wks.merge_cells(
-                    start_row=startrow + cell.row + 1,
+                    startrow=startrow + cell.row + 1,
```

Comment:
Any reason for this? We loosely follow PEP8 guidelines so underscore is desirable here

Reply:
this was unintentional, result of a search and replace.